### ISSUE TITLE
have the website open source so people can create proxies lol

### ISSUE THREAD

**Issue by un-simp** [snipergaming888/snipergaming888.github.io#11]
all  updates could be pushed automagicly through git so you can keep this repo updated for people to use lmao

**Comment by snipergaming888** [comment]
> all updates could be pushed automagicly through git so you can keep this repo updated for people to use lmao

Github slow

**Comment by un-simp** [comment]
Fair enough

**Comment by hdboye** [comment]
its public on github ofc its open source

**Comment by un-simp** [comment]
> 
> 
> its public on github ofc its open source

i mean the website that is being transfered to. as snipergaming has said that new mods wont be pushed here

**Comment by snipergaming888** [comment]
Ima keep these on here, but the new ones will not be on git, cuz I would take to long to upload a new mod twice instead of just doing it one time. I'm not getting rid of this rep.

**Comment by un-simp** [comment]
i just though of this. wouldnt you be able to use github pages and link the domain?

**Comment by snipergaming888** [comment]
> i just though of this. wouldnt you be able to use github pages and link the domain?

Yes but I bought a 100GB server and I wanna use it cuz it has a really good file Manager

**Comment by snipergaming888** [comment]
Accidentally closed it lmao
